nautobot/dcim/filters.py

```diff
@@ -122,6 +122,30 @@ class RegionFilterSet(NautobotFilterSet, NameSlugSearchFilterSet):
         to_field_name="slug",
         label="Parent region (slug)",
     )
+    children_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="children__id",
+        queryset=Region.objects.all(),
+        to_field_name="id",
+        label="Children (ID)",
+    )
+    children = django_filters.ModelMultipleChoiceFilter(
+        field_name="children__slug",
+        queryset=Region.objects.all(),
+        to_field_name="slug",
+        label="Children (slug)",
+    )
```

Comment:
Per https://github.com/nautobot/nautobot/blob/develop/nautobot/docs/development/best-practices.md#mapping-model-fields-to-filters (specifically see "For foreign-key related fields on new core models for v1.4 or later:") our plan for 1.4 and later is to develop a new `NaturalKeyMultipleChoiceFilter` that combines slug filtering and id filtering into a single filter field, and use that for all FK filters, rather than creating two separate filter fields for each FK as has been the existing pattern.

Maybe you get to be the lucky person who implements that Filter class, or maybe you can choose to start with the non-FK fields that need to be added to the DCIM models first. :-)

Reply:
Ha! I'll work with you on this implementation, or yeah, perhaps we should skip the pk fields for the moment since they aren't "easy".

Reply:
`NaturalKeyMultipleChoiceFilter` pushed and ready for review

Reply:
I made another change to this to match existing filters (changed slug to exact instead of icontains)